packages/dd-trace/test/appsec/iast/vulnerability-reporter.spec.js

```diff
@@ -382,6 +392,55 @@ describe('vulnerability-reporter', () => {
       })
     })
 
+    it('should not deduplicate vulnerabilities if not enabled', () => {
+      start({
+        iast: {
+          deduplicationEnabled: false
+        }
+      })
+      const iastContext = { rootSpan: span }
+      addVulnerability(iastContext, createVulnerability('INSECURE_HASHING', { value: 'sha1' }, 888,
+        { path: 'filename.js', line: 88 }))
+      addVulnerability(iastContext, createVulnerability('INSECURE_HASHING', { value: 'sha1' }, 888,
+        { path: 'filename.js', line: 88 }))
+      sendVulnerabilities(iastContext.vulnerabilities, span)
+      expect(span.addTags).to.have.been.calledOnceWithExactly({
+        'manual.keep': 'true',
+        '_dd.iast.json': '{"sources":[],"vulnerabilities":[{"type":"INSECURE_HASHING","hash":3410512691,' +
+          '"evidence":{"value":"sha1"},"location":{"spanId":888,"path":"filename.js","line":88}},' +
+          '{"type":"INSECURE_HASHING","hash":3410512691,"evidence":{"value":"sha1"},"location":' +
+          '{"spanId":888,"path":"filename.js","line":88}}]}'
+      })
+    })
+  })
+
+  describe('clearCache', () => {
+    let span
+    let originalSetInterval
+    let originalClearInterval
+
+    before(() => {
+      originalSetInterval = global.setInterval
```

Comment:
Do you know `sinon.useFakeTimers()`? I discovered recently and it's a way of not messing with global intervals, timeouts, etc